### PR TITLE
Move 'scan_for_plugin_message_in_pwq()` to PluginWorkQueue's test-utils

### PR DESCRIPTION
**Pull request by wimax-grapl**
Also, change the feature to "test-utils"; Depending on another package's "integration_tests" was an antipattern that was causing code to unexpectedly run.

### CODE REVIEW

File: src/rust/plugin-work-queue/src/psql_queue.rs [hunk 1]
```diff
@@ -82,7 +82,7 @@ impl PsqlQueue {
         Ok(Self::new(pool))
     }
 
-    #[instrument(skip(pipeline_message), err)]
+    #[instrument(skip(self, pipeline_message), err)]
```
Comment (wimax-grapl) [inline src/rust/plugin-work-queue/src/psql_queue.rs:85]: an unrelated good change

**Review by jgrillo-grapl — APPROVED** [review]
